chaoshub/cli_overload.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+import json
+
+import click
+
+__all__ = ["run"]
+
+from chaoslib.settings import load_settings
```

Comment:
same comment regarding the imports as before :)

Reply:
Done.

Reply:
You can't specify help on an argument directly. You can make it part of the dostring long `help` entry. In this case it's a simple refinement of the command so it doesn't really warrant that.

In fact the more I consider this the less I like `chaos login hub`, as `hub` isn't really adding anything! I'll remove it entirely.